zdm/orig_tests/test_ics_dmg_mc.py

```diff
@@ -0,0 +1,321 @@
+"""
+Examines the effect of handling of Galactic DM
+Uses Monte Carlo data for ICS generated for
+    DM_MW = 0,100,200,500 pc/cm3
+    (generate using ics_dmg_mc.py)
+Runs agaist H0
+Also outputs component likelihoods (p(z,DM),p(z|DM), etc)
+
+Plots results.
+
+"""
+import pytest
+
+from zdm import io
+from zdm.craco import loading
+from pkg_resources import resource_filename
+import os
+import copy
+import pickle
+import numpy as np
+from astropy.cosmology import Planck18
+
+from zdm import cosmology as cos
+from zdm import misc_functions
+from zdm import parameters
+from zdm import survey
+from zdm import iteration as it
+
+from IPython import embed
+from matplotlib import pyplot as plt
+import matplotlib
+
+defaultsize=14
+ds=4
+font = {'family' : 'normal',
+        'weight' : 'normal',
+        'size'   : defaultsize}
+matplotlib.rc('font', **font)
+
+def make_grids(gamma):
+    if gamma==0:
+        raise ValueError("WARNING: gamma=0 is not currently implemented")
+    ############## Load up ##############
+    input_dict=io.process_jfile('../../papers/H0_I/Analysis/Cubes/craco_H0_Emax_cube.json')
+
+    # Deconstruct the input_dict
+    state_dict, cube_dict, vparam_dict = it.parse_input_dict(input_dict)
+    
+    surveys = []
+    if (state_dict['energy']['luminosity_function'] == 1):
+        names = ['CRAFT_ICS_g1a1_GMG_0','CRAFT_ICS_g1a1_GMG_100',
+        'CRAFT_ICS_g1a1_GMG_200','CRAFT_ICS_g1a1_GMG_500',
+        'CRAFT_ICS_g1a1_GMG_0_200','CRAFT_ICS_g1a1_GMG_0_500',
+        'CRAFT_ICS_g1a1_GMG_500_as_0','CRAFT_ICS_g1a1_GMG_200_as_0','CRAFT_ICS_g1a1_GMG_100_as_0']
```

Comment:
Are these survey files in the repo? They look like they have to be generated, so could you explain what is the difference between files eg CRAFT_ICS_g1a1_GMG_200 and CRAFT_ICS_g1a1_GMG_0_200? @cwjames1983